bep001-naming-conventions.md

```diff
@@ -0,0 +1,73 @@
+Template:
+
+```
+sub-<participant_label>/[ses-<session_label>/]
+    anat/
+        sub-<participant_label>_<modality_label>.json
+        sub-<participant_label>[_ses-<session_label>][_indexable_metadata-<index>][_acq-<label>][_part-<mag/phase>][_ce-<label>][_rec-<label>][_run-<index>]_<modality_label>.nii[.gz]
+        sub-<participant_label>[_ses-<session_label>][_indexable_metadata-<index>][_acq-<label>][_part-<mag/phase>][_ce-<label>][_rec-<label>][_run-<index>]_<modality_label>.json
+```
+
+A naming convention that can logically group parametrically linked anatomy imaging datasets can be achieved by combining following key/value pairs:
+
+1. Suffix (a.k.a modality_label)               (REQUIRED)
+2. [\_indexable\_metadata-<index>]             (RECOMMENDED)
+3. [\_acq-<label>]                             (RECOMMENDED)
+4. [\_part-<mag/phase>]                        (OPTIONAL)
+
+
+### Suffix
+
+Quantitative MRI (qMRI) techniques can be referred by i) sequence names, ii) outputs they generate or iii) special acronyms. As a result, a list of qMRI methods (values) cannot be semantically linked to a single key. To free this field from a key tag, names of the qMRI methods are stored in the `suffix` (a.k.a `<modality_label>`). If several scans of the same modality are intended to create a quantitative map, they MUST be labeled by `suffix`.
+
+| Suffix  | Method Name                              | Output Name                    |
+|---------|------------------------------------------|--------------------------------|
+| VFA     | Variable Flip Angle                      | T1Map                          |
```

Comment:
this is akin to using `FLASH` and can give rise to `T1Map`, `T2*Map`, and `PDMap`

Reply:
This was the reason why instead of using sequence names, method names were suggested in the suffix. For example, T2* mapping with multi-echo `FLASH` for  can be grouped under another label, which would make it easier to signify the purpose of that collection of images. IMO, overloading one label with more than one method is not really helpful for users and validation.